### DISCUSSION TITLE
Need Help About Hugo Permalink And Pagination

### DISCUSSION THREAD

**Discussion by BayuAngora** [community/community#23032] [New to GitHub]
<p>Dear All</p>
<p>I’ve read Hugo documentation and some thread here, but I still get an issue.<br>
<a href="https://angora.me" rel="nofollow noopener">https://angora.me</a></p>
<p>I want to setting my permalink to this:<br>
<a href="http://mysite.netlify.com/my-title/" class="onebox" target="_blank" rel="nofollow noopener">mysite.netlify.com/my-title/</a></p>
<p>Also I want to limit post from default 10 posts per page to only 6 posts per page with this pagination slug on blog:<br>
<a href="http://mysite.netlify.com/blog/" class="inline-onebox-loading" rel="nofollow noopener">mysite.netlify.com/blog/</a> &lt;- 6 posts here<br>
<a href="http://mysite.netlify.com/blog/2/" class="inline-onebox-loading" rel="nofollow noopener">mysite.netlify.com/blog/2/</a> &lt;- 6 posts here<br>
<a href="http://mysite.netlify.com/blog/3/" class="inline-onebox-loading" rel="nofollow noopener">mysite.netlify.com/blog/3/</a> &lt;- 6 posts here</p>
<p>Regards</p>

**Comment by lee-dohm** [comment, marked as answer]
<p>Unfortunately, we don’t support Hugo or Netlify. You may want to check their individual support resources.</p>
<ul>
<li><a href="https://discourse.gohugo.io" rel="nofollow noopener">Hugo Community</a></li>
<li><a href="https://www.netlify.com/docs/" rel="nofollow noopener">Netlify documentation</a></li>
</ul>
<p>I hope that helps!</p>
